dotnet/dex-agent/appsettings.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
nit: inconsistent spacing between `appsettings.Development.json`, and `appsettings.json`